crates/vhost/src/backend.rs

```diff
@@ -57,6 +61,35 @@ impl VringConfigData {
             0
         }
     }
+
+    #[cfg(feature = "vhost-kern")]
```

Comment:
Can we move this changes into `src/vhost_kern/mod.rs`, so we can avoid those condition compilations.

Reply:
I moved the method into `src/vhost_kern/mod.rs`, Please take a look, thanks.